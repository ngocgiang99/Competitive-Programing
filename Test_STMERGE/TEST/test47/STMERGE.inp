8
5 9
614052153 456644404 347248338 782793008 325535047 320248184 350891879 607187469 577875465 
570660919 733564036 523921665 364046134 341716408 162323966 143953520 254840513 537304703 
408184808 742231296 471165664 519458641 380974233 937260949 876278280 993192243 534206118 
929013077 349439214 316285290 253025507 232995237 468661895 375304572 240900709 564943401 
274865672 166837036 904057985 725921479 353876710 880625281 269975757 348529060 914256597 
1 8
829633828 361547212 450939755 873135495 278685165 90510846 246379082 799854688 
5 7
629772129 75731515 505319725 413477930 921830389 137352175 257912366 
547166670 260113804 198693008 139855755 298201090 649120234 974431092 
611903786 361980987 91946259 83891759 457221424 922526766 812618062 
714053271 585144833 748518351 963711263 834442432 126023747 388746581 
110699852 22684415 944395483 896954163 344646872 7697386 894843629 
1 5
702307925 887017951 420812047 514344805 224977237 
10 10
669604272 703033920 269897526 102444554 151880175 839385258 216763301 451275583 340924007 33368681 
647749895 371895500 160173469 752408465 780777987 565193445 236054614 21741687 480417659 674716670 
678354328 695567793 369477783 368175309 705148583 843906067 634494650 43347528 53938439 320758451 
361397642 202593698 336496255 887726821 663328491 367803985 761915667 312725054 67628291 569860618 
175062793 163325588 885606967 417466562 931856100 533330939 830905131 180284077 218609514 745452233 
454815429 418673218 630828449 884558226 145057833 39618613 988474061 787822379 112645310 753881978 
886392947 371204266 18740774 748598087 955543128 170958691 124298825 644815633 378728577 859386091 
814683466 968677518 444240179 543228184 908166543 203407280 727474420 307948543 400683452 580757167 
261872263 545812979 468980407 145998335 840727283 35102603 865598373 690920021 983873404 376634268 
793622506 186655397 439261095 303545146 574069548 632193062 452904249 915637690 362538832 813356107 
3 1
716387450 
68108490 
892633319 
7 6
311214488 736872396 449355163 523999158 477908714 235781037 
809516023 654886227 45035829 848588686 681278366 834209545 
595109166 741336756 580162405 671970880 678319049 530363556 
789287522 839880859 340134973 147826467 336561130 380684800 
51649021 249489463 836282256 532592275 281200832 327382563 
870691116 968913082 758128249 703765358 119637712 282671386 
379624798 214802652 516403923 410845335 985542116 899950450 
1 7
175353536 814934645 932517000 932308927 599198951 626463594 3919908 
