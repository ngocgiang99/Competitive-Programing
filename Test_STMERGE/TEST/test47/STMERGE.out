274865672
336889928
110699852
224977237
33368681
716387450
235781037
3919908
